src/post_processing.jl

```diff
@@ -1054,7 +1055,7 @@ function analyze_and_plot_data(prefix...; run_index=nothing)
         composition, species, collisions,
         geometry, drive_input, num_diss_params, manufactured_solns_input = input
 
-    if !is_1D1V
+    if !is_1D1V || true
```

Comment:
This is an accident -- but it does represent the fact that I do continue to use my 2D diagnostics for 1D1V simulations. I don't know how to extract individuals time slices from a gif, so there are certain plots in the 2D grouping that I continue to want printing out!

Reply:
You won't be surprised that my answer to this is: it's not a problem in `makie_post_processing`! If we wanted to keep `post_processing`, we should really fix it so that plots that make sense in both 1D and 2D are always made, without duplicating code...